packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -266,30 +313,36 @@ class RefreshIndicator extends StatefulWidget {
 
 /// Contains the state for a [RefreshIndicator]. This class can be used to
 /// programmatically show the refresh indicator, see the [show] method.
-class RefreshIndicatorState extends State<RefreshIndicator> with TickerProviderStateMixin<RefreshIndicator> {
+class RefreshIndicatorState extends State<RefreshIndicator>
+    with TickerProviderStateMixin<RefreshIndicator> {
   late AnimationController _positionController;
   late AnimationController _scaleController;
   late Animation<double> _positionFactor;
   late Animation<double> _scaleFactor;
   late Animation<double> _value;
   late Animation<Color?> _valueColor;
 
-  _RefreshIndicatorMode? _mode;
+  RefreshIndicatorStatus? _mode;
   late Future<void> _pendingRefreshFuture;
   bool? _isIndicatorAtTop;
   double? _dragOffset;
-  late Color _effectiveValueColor = widget.color ?? Theme.of(context).colorScheme.primary;
+  late Color _effectiveValueColor =
```

Comment:
Avoid line breaks after assignment operators, as per the style guide: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#prefer-a-maximum-line-length-of-80-characters